### ISSUE TITLE
Weird results on grayscale (triplicated) images

### ISSUE THREAD

**Issue by IliasMAOUDJ** [facebookresearch/detectron2#3298]
Hello everyone,

My training set is composed of synthetic images and my validation set is composed of real images annotated with VGG image annotator.

I trained the model on grayscale images (RGB images with 3 channels set to the same value). When I do the inference on my model and set the PIXEL_MEAN value to [30,30,30] (as an example), the result is really bad. However if I set the second and third channel values something different (for example [30, 180, 180]), the result is way better on the validation set. 

I don't understand what happens here since both my training images and test images are grayscale images, so the pixel_mean should be the same value triplicated. Is there something I missed ? 

Thanks!
